queen-of-versions/build.gradle

```diff
@@ -28,7 +28,7 @@ android {
 
 dependencies {
     api project(path: ':prince-of-versions')
-    implementation 'com.google.android.play:core:1.7.1'
+    implementation 'com.google.android.play:core:1.7.2'
```

Comment:
In the mean time 1.7.3 has been released.
Let's create an issue and a task to change & test that in near future.